### PR TITLE
Change setup path of bck.conf file

### LINKED ISSUE #282: Make default config file as /tmp/bck.conf
It requires sudo privilege to install default config file to /etc - which is not we want.

### PR DESCRIPTION
**Pull request by ziyaddin**
Fixes #282

### CODE REVIEW

**Comment by ShahriyarR** [comment]
Is it complete fix? or you have some ongoing work?

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from setuptools import setup
+from os.path import expanduser
+home = expanduser("~")
 
-datafiles = [('//etc', ['general_conf/bck.conf'])]
+datafiles = [(home+'/.autoxtrabackup', ['general_conf/bck.conf'])]
```
Comment (marmeladze) [inline setup.py]: you should better import `os.path` and, call `os.path.join` with `home` and `.autoxtrabackup` folders.

`os.path.join(home, '.autoxtrabackup')`

always let the language to do the hard work :)

**Comment by ziyaddin** [comment]
@ShahriyarR the current commit fixes the path used in setup phase. I will try to fix all hardcoded paths and push a new commit.

File: setup.py (hunk 1 above)
Comment (ziyaddin) [reply setup.py]: Thanks for suggestion

**Comment by ShahriyarR** [comment]
Merging this one as I need to proceed to fix other related issues.
Thanks for PR.

**Pull request merged**